.storybook/pages/cds-core-shim.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Pages/cds-core shim" />
+
+# @cds/core shim
+
+Clarity provides a theme the enables the `@clr/ui` package to consume the Clarity Core design tokens
+and theme system. Once installed, `@clr/ui` applications will be able to use all Clarity Core
```

Comment:
Should we specify that this shim only applies the colors, and not the entire theme (spacing, typography, etc)?

Reply:
Yeah, I will reword this. I don't even think the typography css is even needed to use the shim. It's more a separate thing.

Reply:
I have updated this page. https://486--storybook-clarity-design.netlify.app/?path=/story/cds-core-shim--page

Reply:
I have updated this page.

- Page: https://486--storybook-clarity-design.netlify.app/?path=/story/cds-core-shim--page
- Changes: https://github.com/vmware-clarity/ng-clarity/compare/b6ff882bee0efcf34692c9a47a1d02f1fe60289b..fc0e47aba5f29b442e0a31695d084974db15e88e